src/test/java/org/opentripplanner/routing/algorithm/TestBikeRental.java

```diff
@@ -1,5 +1,12 @@
 package org.opentripplanner.routing.algorithm;
```

Comment:
Since you're already touching the file, wouldn't the convention be to call it `BikeRentalTest`?

Reply:
I've renamed both tests to `BikeRentalTest` and `CarPickupTest`.